docs/create-packages/Sign-a-Package.md

```diff
@@ -14,7 +14,7 @@ A signed package allows for content integrity verification checks, which provide
 
 ## Get a code signing certificate
 
-Valid certificates can be obtained from a public certificate authority such as [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), or [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml). You can obtain the complete list of certification authorities trusted by Windows from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
+Valid certificates can be obtained from a public certificate authority such as [SSL.com](https://www.ssl.com/certificates/code-signing/), [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), or [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml). You can obtain the complete list of certification authorities trusted by Windows from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
```

Comment:
Duplicate of https://github.com/NuGet/docs.microsoft.com-nuget/pull/2851
Same PR comments regarding alphabetic ordering still applies.

Reply:
the bulleted, alphabetized list can be found here https://github.com/NuGet/docs.microsoft.com-nuget/pull/3046/commits/f4b894ec23586cd5a13a75c5f49d9af4a3348a30